lib/buffy/throttle.ex

```diff
@@ -219,6 +223,16 @@ defmodule Buffy.Throttle do
       @impl GenServer
       @spec handle_info(:timeout, Buffy.Throttle.state()) :: {:stop, :normal, Buffy.Throttle.state()}
       def handle_info(:timeout, {key, args}) do
+        selected_jitter = min(:rand.uniform(unquote(jitter) + 1) - 1, 0)
```

Comment:
Leaving a note: https://github.com/ferd/backoff Since it is `Ferd`, I tend to use it 😄

Reply:
Yeahhh, I went with a super simple jitter implementation instead of adding another library. Trying to keep dependencies down.